### PR TITLE
Added the integration like on mpu6000

### PR DESCRIPTION
**Pull request by davids5**
@LorenzMeier 

This add the integration, 

Please look at a diff between the mpu6000 and this file.

Note the different IOCTL for the HW filter in the 9250 and ho that is done in the mpu6000.

How would you like this reconciled?

### CODE REVIEW

**Comment by LorenzMeier** [comment]
It looks good. But only a soak test will tell. IOCTLs are not of concern for the current integration. Please file a separate issue with direct links to those and more detail, those should not hold up this effort.

File: src/drivers/mpu9250/mpu9250.cpp [hunk 1]
```diff
@@ -1669,6 +1679,14 @@ MPU9250::measure()
...
 	arb.z = _accel_filter_z.apply(z_in_new);
 
+	math::Vector<3> aval(x_in_new, y_in_new, z_in_new);
+	math::Vector<3> aval_integrated;
+
+	bool accel_notify = _accel_int.put(arb.timestamp, aval, aval_integrated, arb.integral_dt);
+	arb.x_integral = aval_integrated(0);
+	arb.y_integral = aval_integrated(1);
+	arb.z_integral = aval_integrated(2);
+
```
Comment (LorenzMeier) [inline src/drivers/mpu9250/mpu9250.cpp]: The integral dt field is not written here.

Comment (davids5) [inline src/drivers/mpu9250/mpu9250.cpp]: I reviewed it and it is an reference output from put
image

Comment (LorenzMeier) [inline src/drivers/mpu9250/mpu9250.cpp]: Sorry, my bad. I forgot how I did it.

**Comment by LorenzMeier** [comment]
Ok, so lets rebase the XRacer branch on this and it should be good to go.

**Pull request merged**